pkgs/applications/science/biology/mosdepth/default.nix

```diff
@@ -17,13 +17,13 @@ let
 
 in stdenv.mkDerivation rec {
   pname = "mosdepth";
-  version = "0.3.1";
+  version = "0.3.2";
 
   src = fetchFromGitHub {
     owner = "brentp";
     repo = "mosdepth";
     rev = "v${version}";
-    sha256 = "1kcrvamrafz1m0s7mlbhaay8jyg97l1w37p6syl36r2m1plmwxjd";
+    sha256 = "sha256-uui4yC7ok+pvbXVKfBVsAarH40fnH4fnP8P4uzOqztQ=";
```

Comment:
Do you have to specify the `sha256-` here if it's already in the `sha256` ?

Reply:
This is the backwards compatible way of using a new format I think we're transitioning to.

Reply:
that is fine for now.